otter_welcome_buddy/cogs/englishclub_reminder.py

```diff
@@ -0,0 +1,147 @@
+import os
+import re
+
+import discord
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from discord import TextChannel
+from discord.ext import commands
+from discord.ext.commands import Bot
+from discord.ext.commands import Context
+
+from otter_welcome_buddy.common.constants import CronExpressions
+from otter_welcome_buddy.common.discord_channel import DiscordChannel
+from otter_welcome_buddy.common.utils.dates import DateUtils
+from otter_welcome_buddy.common.utils.types.common import DiscordChannelType
+from otter_welcome_buddy.formatters import englishclub_message
+
+
+def is_allowed(ctx: Context) -> bool:
+    """
+    Return if a user is allowed to run a command or not
+    """
+    return ctx.author.id == int(os.environ["USER_ONE_ID"]) or ctx.author.id == int(
+        os.environ["USER_TWO_ID"],
+    )
+
+
+class English(commands.Cog):
+    """
+    English command events, where notifications about English sessions
+    Commands:
+        englishclub start:     Start cronjob for timeline messages
+        englishclub stop:      Stop cronjob for timeline messages
+    """
+
+    def __init__(self, bot: Bot, messages_dependency: type[englishclub_message.Formatter]):
+        self.bot: Bot = bot
+        self.messages_formatter: type[englishclub_message.Formatter] = messages_dependency
+        self.scheduler: AsyncIOScheduler = AsyncIOScheduler()
+        self.__configure_scheduler()
+
+    @commands.group(
+        brief="Commands related to English Club!",
+        invoke_without_command=True,
+        pass_context=True,
+    )
+    @commands.check(is_allowed)
```

Comment:
Instead of using this, just check if is a collaborator (aka `OTTER_MODERATOR` or not, that is in constants once you rebase to **develop** branch).
```suggestion
    @commands.has_role(OTTER_MODERATOR)
```
That applies for all of them.